spec/spec_helper.rb

```diff
@@ -109,3 +109,68 @@ def item_from_foxml(foxml, item_class = Dor::Base, other_class = ActiveFedora::O
   end
   result
 end
+
+################################################################
+# Users
+
+def log_in_as_mock_user(subject)
+  user = double(
+    :webauth_user,
+    :login => 'sunetid',
+    :logged_in? => true
+  )
+  allow(subject).to receive(:webauth).and_return(user)
+end
+
+def admin_user
+  webauth = double(
+    'WebAuth',
+    :login => 'sunetid',
+    :attributes => {'DISPLAYNAME' => 'Admin'}
+  )
+  @admin_user = User.find_or_create_by_webauth(webauth)
+  allow(@admin_user).to receive(:groups).and_return(User::ADMIN_GROUPS)
```

Comment:
I did try your suggestion and I don't know why it doesn't work, but it didn't.  So, I'm kinda tired of this PR after working all day on my Sat.  So my mantra now is "If it ain't broke, don't fix it".  If you got a better idea, checkout the branch and try it.  If it works and it's a much better approach, I'm all ears.  FWIW, this consolidation of user mocks into the spec helper has removed a lot of duplication and the weird behavior across feature/integration specs.  It's not applied everywhere, because some specs need very particular user mocks, but most specs just need an `admin_user` to be the `current_user`.  This serves that purpose.